src/Legacy/Storage.php

```diff
@@ -1915,28 +1915,72 @@ private function getFieldName($name)
      */
     private function getEscapedSortorder($name, $prefix = 'r')
     {
-        list($name, $asc) = $this->getSortOrder($name);
+        $order = '';
 
-        // If we don't have a name, we can't determine a sortorder.
-        if (empty($name)) {
-            return false;
-        }
-        if (strpos($name, 'RAND') !== false) {
-            $order = $name;
-        } elseif ($prefix !== false) {
-            $order = $this->app['db']->quoteIdentifier($prefix . '.' . $name);
-        } else {
-            $order = $this->app['db']->quoteIdentifier($name);
-        }
+        $separatedOrders = $this->getOrderBys($name);
+
+        $totalOrderByElements = count($separatedOrders);
 
-        if (! $asc) {
-            $order .= ' DESC';
+        foreach ($separatedOrders as $index => $name) {
+            list($name, $asc) = $this->getSortOrder(trim($name));
+
+            // If we don't have a name, we can't determine a sortorder.
+            if (empty($name)) {
+                return false;
+            }
+            if (strpos($name, 'RAND') !== false) {
+                $order .= $name;
+            } elseif ($prefix !== false) {
+                $order .= $this->app['db']->quoteIdentifier($prefix . '.' . $name);
+            } else {
+                $order .= $this->app['db']->quoteIdentifier($name);
+            }
+
+            if (! $asc) {
+                $order .= ' DESC';
+            }
+
+            if ($this->isNotLastItemInArray($totalOrderByElements, $index))
+                $order .= ',';
         }
 
         return $order;
     }
 
     /**
+     * @param $totalOrderByElements
+     * @param $index
+     * @return bool
+     */
+    protected function isNotLastItemInArray($totalOrderByElements, $index)
+    {
+        return $totalOrderByElements !== ($index+1);
+    }
+
+    /**
+     * @param $order
+     * @return array
+     */
+    protected function getOrderBys($order)
+    {
+        $separatedOrders[] = $order;
+
+        if ($this->isMultiOrderQuery($order))
+            $separatedOrders = explode(",", $order);
+
+        return $separatedOrders;
+    }
+
+    /**
+     * @param $order
+     * @return bool
+     */
+    protected function isMultiOrderQuery($order)
+    {
+        return ( strpos($order, ',') !== FALSE ? TRUE : FALSE );
```

Comment:
Per [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted%2FPSR-2-coding-style-guide.md#25-keywords-and-truefalsenull) the booleans should be lower case.